src/drivers/beanstalk/Queue.php

```diff
@@ -117,6 +103,42 @@ public function status($id)
     }
 
     /**
+     * Removes a job by ID
+     *
+     * @param int $id of a job
+     * @return bool
+     * @throws
```

Comment:
complete return description, also if you add `@throws` it needs to say what it throws and why.